src/libs/UserUtils.ts

```diff
@@ -25,36 +30,26 @@ import * as defaultAvatars from '../components/Icon/DefaultAvatars';
  *          }
  *      }
  * }}
- *
- * @param {Object} loginList
- * @param {Object} loginList.errorFields
- * @returns {Boolean}
  */
-function hasLoginListError(loginList) {
-    return _.some(loginList, (login) => _.some(lodashGet(login, 'errorFields', {}), (field) => !_.isEmpty(field)));
+function hasLoginListError(loginList: Login): boolean {
+    const errorFields = loginList?.errorFields ?? {};
+    return Object.values(errorFields).some((field) => Object.keys(field ?? {}).length > 0);
 }
 
 /**
  * Searches through given loginList for any contact method / login that requires
  * an Info brick road status indicator. Currently this only applies if the user
  * has an unvalidated contact method.
- *
- * @param {Object} loginList
- * @param {String} loginList.validatedDate
- * @returns {Boolean}
  */
-function hasLoginListInfo(loginList) {
-    return _.some(loginList, (login) => _.isEmpty(login.validatedDate));
+function hasLoginListInfo(loginList: Login): boolean {
+    return !loginList.validatedDate;
 }
```

Comment:
This change caused this regression: https://github.com/Expensify/App/issues/28643